Fix XFA links (bug 1735738)

cathydev opened the pull request:
At the end the anchor tag inside the button wasn't the problem here, after comparing the reference file and this one 

> https://bug1716758.bmoattachments.org/attachment.cgi?id=9240360

I realized that in the reference file the anchor tag has an inline display and couldn't receive any height. After changing the display to inline-block it started working but since it receives a 100% height the text wasn't showing properly so I added an absolute position and now the reference file it's working in both Firefox nightly and Chrome.

It also solves a bug in this file 

> https://bug1716758.bmoattachments.org/attachment.cgi?id=9240360

Where the second link in schedule II wasn't working at all, now all links in both files are working just fine.

Thank you @calixteman for explaining me with so much patience!

closes #14139

Snuffleupagus commented:
Can you please provide a bit more detail in the *commit message* as well (since it's currently a single sentence), and not just in the PR description?
Generally it's always a very good idea if it's possible to understand what a patch does and why from the *commit message* too, since that's often most helpful when looking at a patch on the Git command line (e.g. during bisecting).

catherinemds commented on web/xfa_layer_builder.css:
```diff
@@ -221,6 +221,8 @@
 }
 
 .xfaLink {
+  display: inline-block;
```
Yes, as I mentioned I added it because the inline display can't receive any height. When changing the display the link start working and the absolute position it's for the text in the links to show properly.

Commit:
XFA - Fix xfaLink class to make links work (bug 1735738)

There were some links not working in some XFA files,I realized that the anchor tag that contains the link has an inline display and couldn't receive any height, solved this by adding a "position: absolute". Tested with two different files in Firefox Nightly and Chrome and now all links are working perfectly fine. Added reftest to avoid future regressions

calixteman reviewed (approved):
LGTM, thank you for doing this.

calixteman commented:
/botio makeref
